test/graphinfo.jl

```diff
@@ -26,9 +26,9 @@ m = Model(; zip(keys(model), values(model))...) # uses Model(; kwargs...) constr
 # test the type of the model is correct
 @test typeof(m) <: Model
 sorted_vertices = get_sorted_vertices(m)
-@test typeof(m) == Model{Tuple(sorted_vertices)}
+@test typeof(m) <: Model{Tuple(sorted_vertices)}
 @test typeof(m.g) <: GraphInfo <: AbstractModelTrace
-@test typeof(m.g) == GraphInfo{Tuple(sorted_vertices)}
+@test typeof(m.g) <: GraphInfo{Tuple(sorted_vertices)}
```

Comment:
```suggestion
@test m.g isa GraphInfo{Tuple(sorted_vertices)}
```

Reply:
I would have used `isa`, but doesn't work in this case without the full concrete type for the named type les, which is why I used `<:`. Would you prefer I change this to `isa` and use the full types on the rhs?

Reply:
I thought `a isa T` is always equivalent to `typeof(a) <: T`. Can you show me a counterexample?

Reply:
you're right....me being silly again. Fixed...